src/repo.rs

```diff
@@ -2137,7 +2137,7 @@ impl Repository {
     ///
     /// For compatibility with git, the repository is put into a merging state.
     /// Once the commit is done (or if the user wishes to abort), you should
-    /// clear this state by calling cleanup_state().
+    /// clear this state by calling `cleanup_state()`.
```

Comment:
If possible, it's usually nice to make it an intra-doc link.

```suggestion
    /// clear this state by calling [`cleanup_state()`][Repository::cleanup_state].
```